Makefile

```diff
@@ -298,7 +298,7 @@ $(filter-out $(objdir)/uftrace.o, $(UFTRACE_OBJS)): $(objdir)/%.o: $(srcdir)/%.c
 	$(QUIET_CC)$(CC) $(UFTRACE_CFLAGS) -c -o $@ $<
 
 $(objdir)/version.h: PHONY
-	@$(srcdir)/misc/version.sh $@ $(VERSION_GIT) $(ARCH) $(objdir)
+	$(QUIET_GEN)$(srcdir)/misc/version.sh $@ $(VERSION_GIT) $(ARCH) $(objdir)
```

Comment:
I didn't use `QUIET_GEN` here since misc/version.sh prints it inside as it does conditionally.